test/e2e/timeouts.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v3/pkg/appconsts"
+	"github.com/celestiaorg/celestia-app/v3/test/e2e/testnet"
+	"github.com/celestiaorg/celestia-app/v3/test/util/testnode"
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+// VersionedTimeouts runs a simple testnet with 4 validators. It submits both MsgPayForBlobs
+// and MsgSends over 30 seconds and then asserts that at least 10 transactions were
+// committed.
+func VersionedTimeouts(logger *log.Logger) error {
```

Comment:
@staheri14 do we need this test with the modifications to the upgrade test? this seems like we don't need to check that the times were changed twice

Reply:
I agree that extending the major_upgrade_v3 should suffice

Reply:
In the `VersionedTimeouts`, in addition to verifying the overall block time, I was working  on adding a scenario in which we could start one of the nodes later (the reason we have  `indices`  parameter in the methods like `StartNodes` is to control which nodes to start first) and this way we trigger fast sync reactor, and verify if it works properly (however such a test would have been possible with the rpc endpoint which is now deleted).